flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/ScalingMetricCollector.java

```diff
@@ -387,19 +389,18 @@ protected Map<String, FlinkMetric> getFilteredVertexMetricNames(
         }
 
         requiredMetrics.forEach(
-                flinkMetric -> {
-                    filteredMetrics.put(
-                            flinkMetric
-                                    .findAny(allMetricNames)
-                                    .orElseThrow(
-                                            () ->
-                                                    new RuntimeException(
-                                                            "Could not find required metric "
-                                                                    + flinkMetric.name()
-                                                                    + " for "
-                                                                    + jobVertexID)),
-                            flinkMetric);
-                });
+                flinkMetric ->
+                        filteredMetrics.put(
+                                flinkMetric
+                                        .findAny(allMetricNames)
+                                        .orElseThrow(
+                                                () ->
+                                                        new RuntimeException(
+                                                                "Could not find required metric "
+                                                                        + flinkMetric.name()
+                                                                        + " for "
+                                                                        + jobVertexID)),
+                                flinkMetric));
```

Comment:
What is the change here? Indention?

Reply:
yes, I removed some unnecessary curly braces